chronicle-front/src/app/components/viewvideopage/viewvideopage.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Video } from 'src/app/models/Video';
+import { MediaRetrievalService } from 'src/app/services/media-retrieval.service';
+import { VideoPageTransferService } from 'src/app/services/video-page-transfer.service';
+
+@Component({
+  selector: 'app-viewvideopage',
+  templateUrl: './viewvideopage.component.html',
+  styleUrls: ['./viewvideopage.component.css']
+})
+export class ViewvideopageComponent implements OnInit {
+
+
+  @Input() video? : Video;
+
+  constructor(private transfer : VideoPageTransferService, private mediaService : MediaRetrievalService) { }
+
+  ngOnInit(): void {
+    if (this.transfer.video) {
+      this.video = this.transfer.video;
+      this.transfer.video = undefined;
+    } else {
+     this.mediaService.getVideoById(1).subscribe(resp => {
```

Comment:
Did we mean to hard code getting `getVideoById(1)`?

Reply:
Just a test for functionality, will change in the future